packages/cli/src/workflows/workflows.services.ts

```diff
@@ -146,20 +146,17 @@ export class WorkflowsService {
 			relations: config.get('enterprise.features.sharing')
 				? ['tags', 'shared', 'shared.user', 'shared.role']
 				: ['tags'],
+			where: {
+				id: In(sharedWorkflowIds),
+				...filter,
+			},
 		};
 
 		if (config.getEnv('workflowTagsDisabled')) {
```

Comment:
I just realized that this code is broken - we should simply `splice` the `relations` array by removing the `tags` relation, but not removing it completely since, if sharing is enabled, the relations entity contains more than just the tags. 

The current implementation breaks entirely if tags are disabled. Could you also please fix this?